docs/workflows.md

```diff
@@ -0,0 +1,65 @@
+# Workflows for some common use cases 
+
+## 1.- Add a new feature at the end of the feature list.
```

Comment:
Don't add custom numbering on markdown. Avoid any '1.-' and  just use the heading

Reply:
I like numbering :disappointed: 

Done here https://github.com/fluendo/git-upstream-workflow/commit/5c697301c7748ff47fa837edee62234ae0c6c2f7